docs/run.md

```diff
@@ -15,18 +15,6 @@ In Linux, define an alias to simplify the command. If you want it effective ever
 alias lisa="./lisa.sh"
 ```
 
-In Windows Command Prompt, use lisa.cmd to start lisa. Alias definition should be
-
-```bash
-doskey lisa=lisa.cmd
-```
-
-In Windows Powershell,
-
-```bash
-Set-Alias -Name lisa -Value "./lisa.cmd"
-```
-
```

Comment:
You might want to strengthen your git skills too--why the removal of something you did on the commit before?

Reply:
I discussed with Chi, alias is not necessary for Windows so we decided to remove these changes.

Reply:
Awesome! Next time, please git rebase interactive and [ squash commits OR drop bad commit ], instead of proposing changes that cancel each other to be upstreamed :)

Reply:
Thanks for pointing this out. I will pay attention next time.